### DISCUSSION TITLE
We want monitor member's copilot usage. Not only last activity date.

### DISCUSSION THREAD

**Discussion by RichardKCChen-SSSTC** [community/community#57737] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hi,
  Could I have any report or API get my member’s copilot for business usage info such as connection times,duration, when they connect…..,etc.
My boss want to monitor it. Not only last activity info could export from GitHub website currently.
Thanks

**Comment by Akash1134** [comment]
Hi @RichardKCChen-SSSTC ,

These are the product enhancement functionalities we're are aware of and working on. We do not have an ETA about when they will be available and in what form factor. Thanks!

**Reply by askpatrickw** [reply]
this is dupe of https://github.com/orgs/community/discussions/54691 which has more votes.
